src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1333,6 +1328,8 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
         goto err_destroy_dct;
     }
 
+    ucs_assert((init_attr.cq_len[UCT_IB_DIR_TX] / num_dcis) >= self->tx.bb_max);
```

Comment:
plz make it assertv with the corresponding numbers
also it does not take into account number of pools (plz check `tx_cq_size`)

Reply:
when calculating `iface->tx.bb_max` or `cq_len` we don't take into account `num_paths`

Reply:
ok, initially i thought you take the length of created CQ (which takes into count num path), but we do not have it anywhere to check